docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,268 @@
+
+![DALi widget application](./media/dali_widget_application.png)
+
+**Figure: DALi widget application**
+
+For more information on basics of creating a DALi widget application, see [The steps to create a DALi widget application](#create).
+
+<a name="widget-application-api"></a>
+## Widget Application Class
+
+If you want to use DALi for your widget application, you must create the application with the `Dali::WidgetApplication` class. This class provides the means for initializing the resources required by DALi.
+
+The table list the main functions to manage widget instance:
+
+**Table: Main functions**
+
+| Function                             | Description                              |
+|--------------------------------------|------------------------------------------|
+| `mainloop()`                         | The mainloop() function is used to start the event loop. If you do not call the function and start the event loop, DALi cannot call any callback functions for application events.  |
+| `RegisterWidgetCreatingFunction()`   | This function is used to register create function to create widget instance. When widget viewer application requests, then the widget application creates widget instance using the function |
+
+<a name="widget-api"></a>
+## Widget Class
+
+`Dali::Widget` class provides interface for creating custom widget.
+
+The table list the main functions to manage widget instance:
+
+**Table: Main functions**
+
+| Function           | Description                                                                                   |
+|--------------------|-----------------------------------------------------------------------------------------------|
+| `OnCreate()`       | This function is called after the widget instance is created.                                 |
+| `OnTerminate()`    | This function is called after the widget instance is terminated.                              |
+| `OnPause()`        | This function is called when the widget is invisible.                                         |
+| `OnResume()`       | This function is called when the widget is visible.                                           |
+| `OnResize()`       | This function is called before the widget size is changed.                                    |
+| `OnUpdate()`       | This function is called when an event for updating the widget is received.                    |
+| `SetContentInfo()` | This function sends data about current state of widget instance to widget viewer application. |
+
+Most of functions are used for managing widget instance lifecycle. You can describe what will be done on each lifecycle functions.
+
+Widget instance can send data to widget viewer application using `SetContentInfo()` function. If you want to save the current state of widget before deleting it, view the current state of the widget and delete it.
+
+<a name="create"></a>
+## The steps to create a DALi widget application:
```

Comment:
Please change to:

## Create a DALi Widget Application

Reply:
Done